### PR TITLE
Ahmetbibi week2

### PR DESCRIPTION
**Pull request by ahmetbibi**
I couldn't execute trigger with node.js so part2 is not working. But sql querry and trigger is working in `mysql` console. I made an example for Vietnam and if you `insert` 10th language for Vietnam `into` `countrylanguage` table, trigger throws an error message.

### CODE REVIEW

File: Week2/homework/part1/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    });
+
+  program
+    .command('3 <languageName>')
+    .description('to find the number of cities in which a specific language is spoken')
+    .action(languageName => {
+      findCities(languageName);
+    });
+
+  program
```
Comment (saragerion) [inline Week2/homework/part1/index.js]: When I try `node index.js 4A` I get "Please write a number between 1 and 5"  and when I do `node index.js 4` the command is hanging unresponsive.
Can you change this?

File: Week2/homework/part1/queries.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        if (err) throw err;
+        console.log(results);
+      }
+    );
+    connection.end();
+  },
+  listContinents: () => {
+    connection.connect();
+    connection.query(
+      'SELECT DISTINCT Continent, COUNT(Language) AS Language FROM country INNER JOIN countrylanguage ON country.Code = countrylanguage.CountryCode GROUP BY Continent',
```
Comment (saragerion) [inline Week2/homework/part1/queries.js]: Are you sure this is the correct query?
This is returning the following result:
North America : 109,
Asia : 239,
Africa : 310,
Europe : 202,
South America : 42,
Oceania : 82

File: Week2/homework/part2/index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+const queries = require('./queries');
+
+const help = queries.help;
+const addTrigger = queries.addTrigger;
+const insertLanguage = queries.insertLanguage;
+
+if (!process.argv[2]) {
+  help();
+} else {
+  program
```
Comment (saragerion) [inline Week2/homework/part2/index.js]: The commands trigger or insert never get executed. Didn't you forget something? :) 
Tip: you need to parse the arguments...

File: Week2/homework/part2/queries.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  connection.end();
+};
+
+const addTrigger = () => {
+  connection.connect();
+  connection.query(
+    `
+  DROP TRIGGER IF EXISTS language_check;
+  
+  DELIMITER //
```
Comment (saragerion) [inline Week2/homework/part2/queries.js]: I am getting a syntax error if I run the command or I try to run the command via mysql

**Comment by ahmetbibi** [comment]
Hello @saragerion 
Thank you very much for your helpful feedback.
I fixed all the errors. And I hope ERP is true :smiley:  And I found that delimiter is not working with node.js.

**Pull request closed without merging**